kured/templates/daemonset.yaml

```diff
@@ -34,13 +36,18 @@ spec:
             - --lock-annotation=weave.works/kured-node-lock
             - --period=24h
             - --reboot-sentinel=/var/run/reboot-required
-            - --slack-hook-url={{ .Values.slackWebhookUrl }}
+            - --slack-hook-url="$SLACK_WEBHOOK_URL"
             - --slack-username={{ .Values.slackUsername }}
           env:
             - name: KURED_NODE_ID
               valueFrom:
                 fieldRef:
                   fieldPath: spec.nodeName
+            - name: SLACK_WEBHOOK_URL
+              valueFrom:
+                secretKeyRef:
+                  name: kured-values
```

Comment:
should this be called something like `kured-secret` instead?

Reply:
No, unless I change also the secret name in the other repo: https://github.com/hmcts/cnp-flux-config/blob/fd38f02669d62966a524dc2c71aac719e7469942/k8s/aat/kured/creds.yaml